CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 * Add `RSpec/VerifiedDoubleReference` cop. ([@t3h2mas][])
 * Make `RSpec/BeNil` cop configurable with a `be_nil` style and a `be` style. ([@bquorning][])
 * Fix `Capybara/CurrentPathExpectation` autocorrect incompatible with `Style/TrailingCommaInArguments` autocorrect. ([@ydah][])
+* Fix autocorrect for EmptyLineSeparation. ([@johnny-miyake][])
```

Comment:
This should go below "Master".
You'll have to rebase the PR and resolve merge conflicts, as I've just merged another PR. Please accept my apologies for the inconvenience.

Reply:
Sure, I've updated the PR!